packages/beacon-node/src/network/gossip/topic.ts

```diff
@@ -89,7 +89,7 @@ export function getGossipSSZType(topic: GossipTopic) {
     case GossipType.beacon_aggregate_and_proof:
       return sszTypesFor(topic.fork).SignedAggregateAndProof;
     case GossipType.beacon_attestation:
-      return sszTypesFor(topic.fork).Attestation;
+      return isForkPostElectra(topic.fork) ? ssz.electra.SingleAttestation : ssz.phase0.Attestation;
```

Comment:
could also be `return sszTypesFor(topic.fork).SingleAttestation;`?

Reply:
we can add it as an alias though

Reply:
I think the alias makes sense as it matches with what we did in types
https://github.com/ChainSafe/lodestar/blob/48dea55f542b6356d301663ec7c36e5ab9412c1b/packages/types/src/types.ts#L59